From 5414150bde29d7f1432ae91eeb62c5dad8e0e3a2 Mon Sep 17 00:00:00 2001
From: Colin Watson <colin.watson@canonical.com>
Date: Wed, 26 Apr 2017 18:37:29 +0200
Subject: [PATCH 1/7] Copy /cdrom/.disk/info to /var/log/media-info

in order that we get /var/log/installer/media-info on the installed
system; doing this in save-logs is too late because /cdrom is already
unmounted by that point.

Closes: LP: #364649
---
 debian/cdrom-detect.postinst | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/debian/cdrom-detect.postinst b/debian/cdrom-detect.postinst
index 6bf2454..769f527 100755
--- a/debian/cdrom-detect.postinst
+++ b/debian/cdrom-detect.postinst
@@ -85,6 +85,9 @@ mount | grep -q 'on /cdrom' && set_suite_and_codename && exit 0
 if [ -e /cdrom/.disk/info ]; then
 	CDNAME=$(cat /cdrom/.disk/info)
 	log "Detected CD '$CDNAME'"
+	# This produces /var/log/installer/media-info on the installed
+	# system.
+	printf %s "$CDNAME" >/var/log/media-info
 	set_suite_and_codename
 	exit 0
 fi
@@ -265,6 +268,9 @@ fi
 # Install eject-udeb, to be able to use it in the finish-install script.
 anna-install eject-udeb || true
 
+# This produces /var/log/installer/media-info on the installed system.
+printf %s "$CDNAME" >/var/log/media-info
+
 # Hey, we're done
 db_subst cdrom-detect/success cdname "$CDNAME"
 db_input low cdrom-detect/success || [ $? -eq 30 ]
-- 
2.7.4

